CMakeLists.txt

```diff
@@ -228,6 +248,24 @@ if(OPENSN_WITH_LUA)
     )
 endif()
 
+if(OPENSN_WITH_PYTHON)
+    add_subdirectory(python)
+    #    add_subdirectory(test)
+    # 
+    #add_custom_target(test
+    #  COMMAND
+    #  ${CMAKE_BINARY_DIR}/test/opensn-unit &&
+    #          ${CMAKE_SOURCE_DIR}/test/run_tests
+    #          -d ${CMAKE_SOURCE_DIR}/test
+    #          --exe ${CMAKE_BINARY_DIR}/test/opensn-test
+    #          -w3
+    #  WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+    #)
```

Comment:
We probably do not need this commented-out section. When we have testing with python input files, it will be added here...